build/template-restore-build-MSIdentityWeb.yaml

```diff
@@ -11,22 +11,25 @@ steps:
     dotnet workload restore $(IdWebSourceDir)tests\DevApps\blazorwasm-b2c\blazorwasm2-b2c.csproj
   displayName: 'Install wasm-tools'
 
-- task: NuGetAuthenticate@1
-  displayName: NuGet Authenticate
-  inputs:
-      nuGetServiceConnections: 'IDDP Feed'
+# - task: NuGetAuthenticate@1
+#   displayName: NuGet Authenticate
+#   inputs:
+#       nuGetServiceConnections: 'IDDP Feed'
+
+- script: dotnet nuget remove source NuGet
```

Comment:
For this task, ⁠I'm noticing this: 
![image](https://github.com/user-attachments/assets/4bd5b98c-12c3-4a38-a72b-e66b7a19fbc5)
Which is okay, since this task is configured to continue on error, right? Just double checking since I'm used to seeing a green check and not an orange icon:
![image](https://github.com/user-attachments/assets/9d6e1f82-ba0f-4f07-9693-fd9d9b7b5387)

Reply:
thanks @kellyyangsong 
yes, I've noticed as well, and have changed this for a clear way (a small powershell script)

```Powershell
   $nugetSourceIsExternal = (dotnet nuget list source --format Short).Contains("https://api.nuget.org/v3/index.json")
    if ($nugetSourceIsExternal) {
        dotnet nuget remove source NuGet
        dotnet nuget add source https://identitydivision.pkgs.visualstudio.com/_packaging/IDDP/nuget/v3/index.json -n IDDP
    }
```

Reply:
Amazing.. and you came up with this script so quickly. Resolving this thread!